### DISCUSSION TITLE
Rename repo to 'cucumber'

### DISCUSSION THREAD

**Discussion by tyranron** [cucumber-rs/cucumber#131] [Ideas]
@bbqsrc let's rename the GitHub repo just to `cucumber`. I'll update the sources properly, plus will try to reach the author of `cucumber` crate regarding taking its name.

**Comment by bbqsrc** [comment]
Done, and thanks

**Reply by tyranron** [reply]
@bbqsrc there is also a logo for GitHub org and, possibly, crate:
![logo_cucumber_rs_shrinked](https://user-images.githubusercontent.com/7114909/134965146-d0aba72f-f19c-42ed-9c0b-3c36ef7acd5b.png)

**Comment by bbqsrc** [comment]
Cool, added as the org logo :)

**Comment by tyranron** [comment]
@bbqsrc would you be so kind to give me privileges to setup repo's secrets for CI? Or rather, please, create a `CRATESIO_TOKEN` secret for publishing the `cucumber` crate.

Actually, it makes sense to have it as organization level secret.

**Reply by tyranron** [reply]
@bbqsrc ping. We would like to release `0.10.0`. This is the only issue stopping us from it.

**Reply by bbqsrc** [reply]
Hi, sorry, been quite busy. Will get to this in a few hours. Ping me again if I forget.

**Reply by bbqsrc** [reply]
@tyranron I am extremely tired so I won't get to this tonight but I want to have a chat before we do releases, about how we want to manage this project going forward. Would you and @ilslv mind emailing me at brendan@bbqsrc.net and letting me know how we can chat more directly? I use Telegram and Signal if that's useful.

**Reply by tyranron** [reply]
@bbqsrc we can use GitHub for that. Just create a team in our organization and it could have its own private discussions available for team members only. I've bootstrapped the one.

If that is not feasible for some reason, I'd opt-in for a Telegram chat.
